components/eam/src/dynamics/se/stepon.F90

```diff
@@ -543,11 +547,13 @@ subroutine stepon_run3(dtime, cam_out, phys_state, dyn_in, dyn_out)
      endif   
 
    endif   
-
+ 
+ 
    call t_barrierf('sync_dyn_run', mpicom)
    call t_startf ('dyn_run')
    call dyn_run(dyn_out,rc)	
    call t_stopf  ('dyn_run')
+ 
```

Comment:
More whitespace changes. It actually looks like this entire file can be reverted to the version on master.

Reply:
Agreed; this was cleaned and the file no longer has any diff with respect to master